Pipfile

```diff
@@ -42,3 +42,5 @@ pytest-randomly = "==3.2.1"
 
 [packages]
 # we don't specify dependencies for the library here for intallation as per: https://pipenv-fork.readthedocs.io/en/latest/advanced.html#pipfile-vs-setuppy
+requests = "*"
```

Comment:
These should not be listed here. If you do need them for tests then please move them to the dev packages. If you just need them to run an AEA then add them as a dependency of the connection/skill/protocol which requires them.

Reply:
Added to dev-packages. 

Right now it is needed for the connection. With the current implementation, http_client is not generic and is in fact a client for the act-py. We might as well change the names, etc so it doesn't give the impassion that it is a http client for any http server.